spec/fixtures/Puppetfile

```diff
@@ -2,3 +2,5 @@ mod 'homebrew',    '0.0.16', :github_tarball => 'boxen/puppet-homebrew'
 mod 'module-data', '0.0.1',  :github_tarball => 'ripienaar/puppet-module-data'
 mod 'stdlib',      '4.1.0',  :github_tarball => 'puppetlabs/puppetlabs-stdlib'
 mod 'sudo',        '0.0.1',  :github_tarball => 'boxen/puppet-sudo'
+mod 'dotfiles',    '0.1.1.toolbear1',  :github_tarball => 'toolbear/puppet-dotfiles'
```

Comment:
Change to `jamieconnolly/puppet-dotfiles` once jamieconnolly/puppet-dotfiles#1 merged.